.github/workflows/release.yml

```diff
@@ -54,4 +54,8 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
-        gh release create $ORT_SERVER_VERSION --notes-file RELEASE_NOTES.md
+        PRERELEASE_ARG=""
```

Comment:
Nit: You could even omit this, as interpolating a non-existing variable will also result in the empty string.